src/models/List.ts

```diff
@@ -32,4 +32,7 @@ export class List {
         this.ingredients.splice(ingredientIndex, 1);
     }
 
+    setListName(newName:string):void{
```

Comment:
Add spaces to be consistent with other parts of the code.
`(newName: string): void {`